examples/basic.js

```diff
@@ -1,12 +1,11 @@
 var EddystoneBeaconScanner = require('../'); // use require('eddystone-beacon-scanner'), if installed from npm
 
-EddystoneBeaconScanner.on('discover', function(urlBeacon) {
-  console.log('discovered Eddystone beacon:');
-  console.log('  url      = ' + urlBeacon.url);
-  console.log('  type     = ' + urlBeacon.type);
-  console.log('  TX power = ' + urlBeacon.txPower);
-  console.log('  RSSI     = ' + urlBeacon.rssi);
-  console.log();
+EddystoneBeaconScanner.on('didFindBeacon', function(beacon) {
```

Comment:
What do you think about using less iOS like delegate names for the events? I liked ```discover``` but open to alternatives.